CME/dataobjects/CMEProvider.php

```diff
@@ -67,6 +68,27 @@ public function loadByShortname($shortname)
 	}
 
 	// }}}
+	// {{{ public function getCreditTitle()
+
+	public function getCreditTitle($hours, $credit_count = 1, $is_free = false)
+	{
+		$locale = SwatI18NLocale::get();
+		return sprintf(
+			SwatString::minimizeEntities(
+				CME::_('%s%s %s%s%s certified by %s')
+			),
+			SwatString::minimizeEntities($locale->formatNumber($hours)),
+			$is_free ? ' '.CME::_('Free') : '',
+			'<em>',
+			(abs($hours - 1.0) < 0.01)
+				? SwatString::minimizeEntities($this->credit_title)
+				: SwatString::minimizeEntities($this->credit_title_plural),
+			'</em>',
```

Comment:
These italics are tricky - only `AMA PRA Category 1 Credit(s)™` should be italicized, and not the AAFP credits. I wonder if we need a `getFormattedTitle()` to add or not add the italics.